test/validator/analyze_test.dart

```diff
@@ -131,7 +131,7 @@ void main() {
     await expectValidation(
       error: allOf([
         contains('`dart analyze` found the following issue(s):'),
-        contains('Analyzing lib, bin, pubspec.yaml...'),
+        contains('Analyzing bin, lib, pubspec.yaml...'),
```

Comment:
There is a test of catching errors in a pubspec.yaml. Could you make one (or extend that one) to test that we actually analyze these files?